dd-sdk-android/apiSurface

```diff
@@ -225,14 +225,21 @@ abstract class com.datadog.android.rum.tracking.ActivityLifecycleTrackingStrateg
   companion object 
 class com.datadog.android.rum.tracking.ActivityViewTrackingStrategy : ActivityLifecycleTrackingStrategy, ViewTrackingStrategy
   constructor(Boolean, ComponentPredicate<android.app.Activity> = AcceptAllActivities())
+  override fun onActivityCreated(android.app.Activity, android.os.Bundle?)
+  override fun onActivityStarted(android.app.Activity)
   override fun onActivityResumed(android.app.Activity)
+  override fun onActivityPostResumed(android.app.Activity)
   override fun onActivityPaused(android.app.Activity)
+  override fun onActivityDestroyed(android.app.Activity)
 interface com.datadog.android.rum.tracking.ComponentPredicate<T>
   fun accept(T): Boolean
-class com.datadog.android.rum.tracking.FragmentViewTrackingStrategy : ActivityLifecycleTrackingStrategy, ViewTrackingStrategy
+class com.datadog.android.rum.tracking.FragmentViewTrackingStrategy : ActivityLifecycleTrackingStrategy, ViewTrackingStrategy, com.datadog.android.rum.internal.tracking.RumEventListener
```

Comment:
In theory a public class should not expose method from an internal interface

Reply:
I can fix that